server_test.go

```diff
@@ -155,6 +158,12 @@ func TestServer(t *testing.T) {
 		assert.Eq(result, "Danny")
 	})
 
+	t.Run("EndWithSlash", func(t *testing.T) {
+		result, _, err := response("GET", ts.URL, "/test/end-with-slash")
```

Comment:
BTW, does go `net/http` cut `/` at the end?

Reply:
It won't, but at line 32 to 36 of router.go will filter empty string, so won't cause a problem